### ISSUE TITLE
Make AWS Command Document Parameters dynamic

### ISSUE THREAD

**Issue by Newbie112222** [awsdocs/aws-systems-manager-user-guide#78]
Hi All,
I am running into an issue where I need to make a variable dynamic in my command document. I already have created a parameter for the same but I want to integrate this in a pipeline and hence I don't want to go in and change the name of the parameter for every run and create a new version of the document.

Any ideas as to how I can make the value of the parameter dynamic?? Or maybe append the code build id or some other param in the build spec on code build?
FYI- I have tried appending $codebuild id etc in build spec but nothing seems to work.

Any help would be appreciated. Thanks!!
Basically I am trying to find a way to make the below parameter Test3 dynamic ex. test1_blah, test2_blah1 etc. but I don't want to create new version of the document to do that. Thanks!!

"schemaVersion": "2.2",
"description": "Command Document ",
"parameters": {
"FolderName": {
"type": "String",
"description": "This folder is where Report will be stored",
"default": "Test3"
}
},

**Comment by mark-amzn** [comment]
We're not aware of easy ways to accomplish this. We're also not certain these methods would work, but if using AWS CodePipeline, perhaps a final pipeline action invokes a Lambda function that updates the Systems Manager parameter value by 1, or a CloudWatch Events event: when the pipeline succeeds, it triggers execution of a Run Command document with a script to update the parameter value by 1. Note: These aren't tested, supported, or verified methods, just some ideas to look into. 

If this is still a need, you might try asking in the Systems Manager developer form (https://forums.aws.amazon.com/forum.jspa?forumID=185).
